st2common/st2common/util/pack.py

```diff
@@ -107,9 +107,10 @@ def validate_config_against_schema(config_schema, config_object, config_path,
     for key in config_object:
         if (jinja_utils.is_jinja_expression(value=config_object.get(key)) and
                 "decrypt_kv" in config_object.get(key) and config_schema.get(key).get('secret')):
-            raise ValueValidationException('Validation Error: decrypt_kv jinja filter'
-                                           ' specified for auto decrypted fields marked'
-                                           ' with `secret: True`')
+            raise ValueValidationException('Values specified as `secret: True` in config schema '
+                                           'are automatically decrypted by default. Use of '
+                                           '`decrypt_kv` jinja filter is not allowed for such '
+                                           'values')
```

Comment:
Why do you put use "`" around secret: True?  Does this make the text highlighted in st2web?  afaik, this doesn't do anything special on the CLI or log.  Also, please add a period "." at the end of the 2nd sentence.

Reply:
Done